src/main/java/org/zalando/nakadi/ShutdownHooks.java

```diff
@@ -29,7 +29,7 @@ private static void onNodeShutdown() {
                 try {
                     hook.run();
                 } catch (final RuntimeException ex) {
-                    LOG.warn("Failed to call on shutdown hook for {}", hook, ex);
+                    LOG.warn("Failed to call on shutdown hook for {}: {}", hook, ex.getMessage());
```

Comment:
It wont't be possible to debug the error if `Runnable` failed and there is only message: `java.lang.NullPointerException`.